sources/packages/backend/libs/integrations/src/services/student-loan-balance/student-loan-balance.service.ts

```diff
@@ -0,0 +1,79 @@
+import { Injectable } from "@nestjs/common";
+import { SystemUsersService } from "@sims/services";
+import { StudentLoanBalance } from "@sims/sims-db";
+import { EntityManager } from "typeorm";
+
+@Injectable()
+export class StudentLoanBalanceService {
+  constructor(private readonly systemUserService: SystemUsersService) {}
```

Comment:
Not related to this PR. But the one which is inserting records from file into the database. Why did we use access the repo and did the inserts directly from processing service instead of calling a service like this ??

Reply:
The `studentLoanBalancesRepo` is the one from the active transaction where the only command executed against it is the `insert`. It is following what we have been doing all across the application when transactions are needed.
Sometimes we do send the `entityManager` over a parameter but in general, for simple commands like that one we just call the insert from the repo from the transition.
Maybe there is confusion because the name `studentLoanBalancesRepo` would be the same as declared in the constructor.
Please let me know if further clarification would be needed.